### DISCUSSION TITLE
Latest Unifi Portal update

### DISCUSSION THREAD

**Discussion by Dudleydogg** [hassio-addons/addon-unifi#366] [Q&A] [answered]
I have some devices I just cannot update due to the portal being out of date anyway to update to latest with this plugin, can I some how download the firmware and update?
thanks

**Comment by frenck** [comment]
Wat do you mean by "portal"?

This add-on runs the UniFi Network Application 7.1.66, which is the newest version available.

**Comment by Dudleydogg** [comment]
Thank you so much for replying, I use many of your plugins.

 

Maybe I have some misunderstanding my Plugin says 

7.0.23 (Build: atag_7.0.23_17288)

And its constantly nagging  me to update to 7.1.66

 

So maybe im doing something wrong how to I trigger the plugin to update itself?

 



 

Astronomy & Computer Consulting

STEPHEN GRIFFITHS    Astronomy | Astrophotography Club 

 ***@***.***> ***@***.*** | <https://www.astrocomputer.net> https://www.astrocomputer.net

**Comment by frenck** [comment, marked as answer]
> So maybe im doing something wrong how to I trigger the plugin to update itself?

Maybe you have lost the add-on repository? I know there is a bug currently in Supervisor that causes this at times (solution is in the making).

When its missing, it will also not offer updates anymore. You can re-add it using this button:

[![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://my.home-assistant.io/badges/supervisor_add_addon_repository.svg)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository)

If that is added, a normal add-on update in Home Assistant should be offered.

**Comment by Dudleydogg** [comment]
Apologies, My Community Repository was broken, so I did not see the update.

Thank you for all your contributions I use many.

Everything is working again.

 

 

 

 

 

Astronomy & Computer Consulting

STEPHEN GRIFFITHS    Astronomy | Astrophotography Club 

 ***@***.***> ***@***.*** | <https://www.astrocomputer.net> https://www.astrocomputer.net

**Comment by frenck** [comment]
> Everything is working again.

Good to hear 👍 

> Thank you for all your contributions I use many.

Your welcome ❤️

**Comment by Dudleydogg** [comment]
That was the exact problem, but it would error each time I tried to readd,  I just build new home assistant, restore with backup and now it all works.

 

Thank you very much.

 

 

 

 

Astronomy & Computer Consulting

STEPHEN GRIFFITHS    Astronomy | Astrophotography Club 

 ***@***.***> ***@***.*** | <https://www.astrocomputer.net> https://www.astrocomputer.net
